src/blood-center/models/blood-center.entity.ts

```diff
@@ -0,0 +1,52 @@
+import { Column, Entity, OneToMany, PrimaryGeneratedColumn } from 'typeorm';
+import { BloodCenterDetailsEntity as BloodCenterDetails } from './blood-center-details.entity';
+import { BloodCenterEventEntity as BloodCenterEvent } from './blood-center-event.entity';
+
+@Entity('blood_centers')
+export class BloodCenterEntity {
```

Comment:
Is that required for the map? Will the lambda function be able to provide that from the scraper?

Reply:
I'm not sure yet how we'll query a maps API for data, but the best bet is pointing the exact coordinates.

If you mean the blood banks scraper – its purpose is to fetch latest blood types state from a given set of blood banks. It will not include their coordinates, since we will be importing them from a Google My Maps map.

Reply:
What about scraping the addresses from [krew.info](https://krew.info/rckik/index.html) ?

Reply:
krew.info doesn't contain those addresses, only an [index page with links to respective blood centers](https://krew.info/rckik/index.html).

I guess you meant scraping them from the [Google My Maps map](https://www.google.com/maps/d/u/0/viewer?mid=142nqMIgmK3tCarABCmdvq8MHGlNZZPMB&ll=52.0339097532797%2C18.934507500000002&z=6)?

Reply:
> I guess you meant scraping them from the [Google My Maps map](https://www.google.com/maps/d/u/0/viewer?mid=142nqMIgmK3tCarABCmdvq8MHGlNZZPMB&ll=52.0339097532797%2C18.934507500000002&z=6)?
My bad, that's exactly what I meant.

I'll add the geolocation data in the upcoming revision,